exatomic/core/gradient.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# Copyright (c) 2015-2018, Exa Analytics Development Team
+# Distributed under the terms of the Apache License 2.0
+
+from exa import DataFrame
+import numpy as np
+import pandas as pd
+
+class Gradient(DataFrame):
+    """
+    The gradient dataframe
+    """
+    # simple function that will have to be seen if it can have any other functions
+    _index = 'gradient'
+    _columns = ['Z', 'label', 'fx', 'fy', 'fz', 'symbol', 'frame']
```

Comment:
To have this dataframe link to a specific atom, columns must require the atom index (`atom`)

Reply:
Whoops I forgot to add that. I actually ended up adding it in the code for the parse_gradient. I'll make sure to add that in.

Reply:
I did add it in actually with the label column. I'm not sure if we should consider re-doing the columns to have adx be the atom index as opposed to label.

Reply:
Label isn't an index because an index must be a unique identifier. Labels repeat for universes that have more than one frame. If you look at the machinery of how universe dataframes are related (I think .network() will show that) you'll see which fields are indexes and their names.

Reply:
replaced atom with label for the next push